app/services/instrumentation.rb

```diff
@@ -0,0 +1,34 @@
+module Instrumentation
+  class << self
+    def append_to_log(args = {})
+      RequestStore.store[:custom_log_items] ||= {}
+      RequestStore.store[:custom_log_items].merge!(args) unless args.blank?
+      RequestStore.store[:custom_log_items]
+    end
+    alias_method :custom_log_items, :append_to_log
```

Comment:
I agree, `args` is not very revealing.

However you're using `custom_log_items` for reading – see `payload[:custom_log_items] = Instrumentation.custom_log_items`